src/util/DateUtils.ts

```diff
@@ -174,6 +174,14 @@ export class DateUtils {
         return value;
     }
 
+    static simpleJsonToString(value: any): string {
```

Comment:
Date Utils? 😆

Reply:
@19majkel94 I looked at simpleArrayToString, discovered that it was odd for it to be in DateUtils, but well, my PR is as simple as it gets, so I did the same thing 😢

Reply:
Ok, so we will leave it for now and later it would be refactored 😉